examples/darksage.py

```diff
@@ -3790,7 +4032,7 @@ def iterate_trees(self):
         from_file_dtype = np.dtype(ordered_dtype, align=True)
         print("from file type = {0}".format(from_file_dtype))
         print("sizeof(file_dtype) = {0}".format(from_file_dtype.itemsize))
-        assert from_file_dtype.itemsize == 1544, "Size of datatypes do not match"
+#        assert from_file_dtype.itemsize == 1544, "Size of datatypes do not match"
```

Comment:
I would recommend keeping the check in - but adjust the known size based on the version of DarkSage being converted